index.rst

```diff
@@ -544,6 +544,13 @@ What's new in PyTorch tutorials?
    :link: advanced/static_quantization_tutorial.html
    :tags: Quantization
 
+.. customcarditem::
```

Comment:
please add to the https://github.com/pytorch/tutorials/blob/main/prototype_source/prototype_index.rst rather than in index.rst